pkg/proxy/ipvs/proxier.go

```diff
@@ -362,10 +362,14 @@ func NewProxier(ipt utiliptables.Interface,
 		}
 	}
 
-	// Set the connection reuse mode
-	if val, _ := sysctl.GetSysctl(sysctlConnReuse); val != 0 {
-		if err := sysctl.SetSysctl(sysctlConnReuse, 0); err != nil {
-			return nil, fmt.Errorf("can't set sysctl %s: %v", sysctlConnReuse, err)
+	if kernelVersion.LessThan(version.MustParseGeneric("4.19")) {
+		return nil, fmt.Errorf("can't set sysctl %s: %v", sysctlConnReuse)
```

Comment:
Not sure we should exit here, just log a warning?

Reply:
Yes we should try and set the other sysctls
If we end up having other restrictions for them we'll list the requirements for all sysctls and decide (some sysctls improve performances and could be considered "not required", some other a really required to make ipvs mode work fine)